### DISCUSSION TITLE
Problem of excessive bandwidth consumption on vercel when hosting my nextjs application

### DISCUSSION THREAD

**Discussion by JuniorHoms** [vercel/community#4452] [Help]
### Summary

It's been less than a month since our site was weeded on vercel but we've noticed an exaggerated consumption of bandwidth (more than 500GB have been consumed).
We've tried everything but we still can't work out what's causing it.
what can we do to resolve this exaggerated bandwidth consumption on vercel?

### Steps to Reproduce

It's been less than a month since our site was weeded on vercel but we've noticed an exaggerated consumption of bandwidth (more than 500GB have been consumed).
We've tried everything but we still can't work out what's causing it.
what can we do to resolve this exaggerated bandwidth consumption on vercel?

**Comment by z4nr34l** [comment]
You need to check what you are serving from Vercel, in most cases such bandwidth is caused by not optimized large images, videos or files served as raw files (PDF, DOC, MP4 etc.)

If you will still have this problem, but can add me to repository - just do it and ping me so I will be able to check and help you in this case ;)

**Reply by Diwannist** [reply]
What do you do instead of raw files; compressed files; can you give a little bit more detail please?
